alluxio-ui/common/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+import {faCheckSquare, faSquare} from '@fortawesome/free-regular-svg-icons'
```

Comment:
A general question about the new `tsx` files - is there usually any code documentation associated with them (similar to Javadocs)?

Reply:
Yes, we can add this (https://github.com/Microsoft/TypeScript/wiki/JSDoc-support-in-JavaScript).